conf/mainnet/Config-with-bitcoin-flag.toml.sample

```diff
@@ -0,0 +1,19 @@
+[node]
+working_dir = "/root/stacks-blockchain/data"
+rpc_bind = "0.0.0.0:20443"
+p2p_bind = "0.0.0.0:20444"
+bootstrap_node = "02da7a464ac770ae8337a343670778b93410f2f3fef6bea98dd1c3e9224459d36b@seed-0.mainnet.stacks.co:20444,02afeae522aab5f8c99a00ddf75fbcb4a641e052dd48836408d9cf437344b63516@seed-1.mainnet.stacks.co:20444,03652212ea76be0ed4cd83a25c06e57819993029a7b9999f7d63c36340b34a4e62@seed-2.mainnet.stacks.co:20444"
+
+[[events_observer]]
+endpoint = "stacks-blockchain-api:3700"
+retry_count = 255
+events_keys = ["*"]
+
+[burnchain]
```

Comment:
this model isn't ideal since it requires hardcoding values that could change

Reply:
Let's talk about this in a call.
It's a sample file with defaults that the user can change if he wishes but the script will never modify it.